ci/dra_aarch64.sh

```diff
@@ -24,4 +24,10 @@ docker save docker.elastic.co/logstash/logstash-oss:${STACK_VERSION}-SNAPSHOT |
 docker save docker.elastic.co/logstash/logstash-ubi8:${STACK_VERSION}-SNAPSHOT | gzip -c > build/logstash-ubi8-${STACK_VERSION}-docker-image-aarch64.tar.gz
 
 echo "GENERATED ARTIFACTS"
-for file in build/logstash-*; do shasum $file;done
\ No newline at end of file
+for file in build/logstash-*; do shasum $file;done
+
+echo "UPLOADING TO INTERMEDIATE BUCKET"
+# Note the deb, rpm tar.gz AARCH64 files generated has already been loaded by the dra_x86_64.sh
```

Comment:
How do we guarantee that the `dra_x86_64.sh` file has been run first? Is there an indication to the user if it has not been done?